openhands/llm/llm.py

```diff
@@ -200,7 +205,40 @@ def wrapper(*args, **kwargs):
 
             try:
                 # we don't support streaming here, thus we get a ModelResponse
-                resp: ModelResponse = self._completion_unwrapped(*args, **kwargs)
+                try:
+                    resp: ModelResponse = self._completion_unwrapped(*args, **kwargs)
+                except RateLimitError as e:
```

Comment:
Just a small thing here, there already is a try / except, it seems we're adding another? Or does github interface fool me 😅

Reply:
Haha yeah there is already a try/except; I was experimenting with OH and wanted to push it beyond its comfort zone to see if it implement weird things 😆